product_default_image/image_constants.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+# From product.template
+# image_type field
+TYPES = {
```

Comment:
ah okay. I saw in the medical constants, integers were being used so that's why I did that here. Makes sense though, I was having to refer back to the file a lot.